docs/guide/limitation_reference.md

```diff
@@ -402,6 +402,25 @@ is evaluated.
 For more information on how to restrict User's access to part of the Subtree 
 follow [the example in the Admin management section](admin_panel.md#restrict-editing-to-part-of-the-tree).
 
+## Version Lock Limitation
+
+A Limitation to specify whether the User can perform edit or unlock actions on 
```

Comment:
Limitations purpose is to limit any actions, not just edit and unlock.

Indirectly it is, but just because it is by default enabled in content/edit and content/unlock policies. Dev can add a limitation to content/read or something else and it will still work just fine.